src/EVM/Types.hs

```diff
@@ -464,14 +466,22 @@ instance Show ByteStringS where
       fromBinary =
         Text.decodeUtf8 . toStrict . toLazyByteString . byteStringHex
 
-instance JSON.ToJSON ByteStringS where
-  toJSON = JSON.String . Text.pack . show
+instance JSON.FromJSON ByteString where
```

Comment:
I'm a little nervous that we will have issues related to [orphan instances](https://wiki.haskell.org/Orphan_instance) if one of our deps defines the same instance. Would it be very annoying to use the `ByteStringS` newtype above for json serialization of transactions?

Reply:
This has now been fixed :) Thanks for pointing this out! :)